ci/gpu/build.sh

```diff
@@ -72,6 +75,9 @@ $WORKSPACE/build.sh
 if hasArg --skip-tests; then
     logger "Skipping Tests..."
 else
+    INSTALL_PREFIX=${INSTALL_PREFIX:=${PREFIX:=${CONDA_PREFIX}}}
+    export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:$INSTALL_PREFIX/lib
```

Comment:
I don't believe these need to be here. This else block is just running the tests, they should've already been installed previously with the build.sh command.

Reply:
the tests wont run properly without these environment variables